Source/ARTRealtime+Private.h

```diff
@@ -113,6 +116,10 @@ NS_ASSUME_NONNULL_BEGIN
 // Message sending
 - (void)send:(ARTProtocolMessage *)msg sentCallback:(nullable ARTCallback)sentCallback ackCallback:(nullable ARTStatusCallback)ackCallback;
 
++ (double)backoffCoefficientWithNumber:(NSInteger)number;
```

Comment:
I'm not sure whether `ARTRealtimeInternal` is the right place to put this. Given that the functionality is also used for deciding channel retry logic, the choice feels a bit arbitrary. I wonder whether it might be better to introduce a new utility class to contain this timeout calculation logic.

Reply:
Also, I think the naming could be clearer:

```suggestion
+ (double)backoffCoefficientForRetryNumber:(NSInteger)retryNumber;
```

Reply:
https://github.com/ably/ably-cocoa/pull/1520/commits/171efb801fb805bb3d47064c713221affaf6405a

Reply:
>a new utility class to contain this timeout calculation logic
https://github.com/ably/ably-cocoa/pull/1520/commits/c11f2b2e6890338d8f233d27c000241ce47f0812
https://github.com/ably/ably-cocoa/pull/1520/commits/fe3139983f2d0d2bb0502c118deb37889e5e268a